build.gradle.kts

```diff
@@ -21,26 +24,31 @@ dependencies {
   implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-csv")
 
   implementation("org.apache.commons:commons-text:1.10.0")
-  implementation("org.springdoc:springdoc-openapi-webmvc-core:1.6.13")
-  implementation("org.springdoc:springdoc-openapi-ui:1.6.13")
-  implementation("org.springdoc:springdoc-openapi-kotlin:1.6.13")
+  implementation("org.springdoc:springdoc-openapi-webmvc-core:1.6.14")
+  implementation("org.springdoc:springdoc-openapi-ui:1.6.14")
+  implementation("org.springdoc:springdoc-openapi-kotlin:1.6.14")
+  implementation("com.microsoft.azure:applicationinsights-web:3.4.7")
+  implementation("commons-codec:commons-codec:1.15")
 
-  runtimeOnly("com.h2database:h2:2.1.214")
   runtimeOnly("org.flywaydb:flyway-core")
   runtimeOnly("org.postgresql:postgresql:42.5.1")
+  testRuntimeOnly("com.h2database:h2:2.1.214")
 
-  testImplementation("io.jsonwebtoken:jjwt:0.9.1")
+  testImplementation("io.jsonwebtoken:jjwt-api:0.11.5")
+  testImplementation("io.jsonwebtoken:jjwt-impl:0.11.5")
+  testImplementation("io.jsonwebtoken:jjwt-orgjson:0.11.5")
   testImplementation("com.github.tomakehurst:wiremock-standalone:2.27.2")
```

Comment:
old wiremock - new is
```
   testImplementation("com.github.tomakehurst:wiremock-jre8-standalone:2.35.0")
```

Reply:
if you run my script in https://github.com/ministryofjustice/hmpps-tech-docs/blob/pgp-SDI-477-spring-boot-3/scripts/upgrade-spring-boot-3.bash it will do it automatically (and other changes too)

Reply:
sorted

Reply:
not sure how - but you appear to have both wiremocks in there now?

Reply:
it is probably because of 
  testImplementation("com.github.tomakehurst:wiremock-jre8-standalone:2.35.0")
  testImplementation("com.github.tomakehurst:wiremock-standalone:2.27.2")

I removed older version